src/patternfly/components/Card/card.scss

```diff
@@ -21,21 +22,49 @@
       box-shadow: var(--pf-c-card--m-hoverable--hover--BoxShadow);
     }
   }
+}
 
-  .pf-c-card__header,
-  .pf-c-card__body,
-  .pf-c-card__footer {
-    flex: 0 0 auto;
-    padding-right: var(--pf-c-card--child--PaddingRight);
-    padding-bottom: var(--pf-c-card--child--PaddingBottom);
-    padding-left: var(--pf-c-card--child--PaddingLeft);
+.pf-c-card__head {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
```

Comment:
If there will only be actions and/or 1 other element in this element, can you just use `align-items: flex-start` here, and remove that from `.pf-c-card__actions`?

Reply:
yes great idea

Reply:
actually when I do this, the text in card head is aligning to the top, and not in the center with the buttons 
<img width="629" alt="Screen Shot 2019-05-13 at 11 03 14 AM" src="https://user-images.githubusercontent.com/20118816/57632095-cf859000-756e-11e9-8d4e-a409c1acd91f.png">